### PR TITLE
[HTM-838] Add delete feature option for edit component

### PR DESCRIPTION
**Pull request by mprins**

### CODE REVIEW

File: projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts [hunk 1]
```diff
@@ -123,6 +123,32 @@ export class EditDialogComponent implements OnInit {
...
+          return this.editFeatureService.deleteFeature$( viewerId, layerId, {
+            __fid: currentFeature.feature.__fid,
+            attributes: currentFeature.feature.attributes,
+          });
+        }),
+      )
+      .subscribe(succes => {
+        if (succes) {
+          this.closeDialog();
+          this.mapService.getLayerManager$()
```
Comment (geertplaisier) [inline projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts:143]: Maybe we can add this as 'shortcut' inside the MapService so its easier to use, something like 
`mapService.refreshLayer(layerId)` where the MapService then executes this code. This is probably something we do more often in the future.

File: projects/api/src/lib/services/tailormap-api-v1.service.ts [hunk 2]
```diff
@@ -90,11 +90,17 @@ export class TailormapApiV1Service implements TailormapApiV1ServiceModel {
...
-      `${TailormapApiConstants.BASE_URL}/${params.applicationId}/layer/${params.layerId}/edit/feature/${params.feature.__fid}`,
-    );
-  }
+    public deleteFeature$(params: { applicationId: string; layerId: string; feature: FeatureModel }): Observable<HttpStatusCode> {
+        return this.httpClient.delete<HttpResponse<Response>>(
+            `${TailormapApiConstants.BASE_URL}/${params.applicationId}/layer/${params.layerId}/edit/feature/${params.feature.__fid}`,
+            { observe: 'response' },
+        ).pipe(
+            map(response => {
+                    return response.status as HttpStatusCode;
```
Comment (geertplaisier) [inline projects/api/src/lib/services/tailormap-api-v1.service.ts:99]: `... as HttpStatusCode` is probably an uneeded cast, `.status` should already be this type?

**Review by geertplaisier — CHANGES_REQUESTED** [review]

File: projects/api/src/lib/services/tailormap-api-v1.service.ts (hunk 2 above)
Comment (mprins) [reply projects/api/src/lib/services/tailormap-api-v1.service.ts:99]: `HttpResponseBase.status` is a number

File: projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts (hunk 1 above)
Comment (mprins) [reply projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts]: ✅

**Pull request merged**